Vagrantfile

```diff
@@ -141,7 +143,17 @@ Vagrant.configure(2) do |config|
     c.vm.network :forwarded_port, guest: 22,   host: 2226,  id: "ssh", auto_correct: true
 
     c.vm.provision "shell", inline: $unixProvision, privileged: false
-    c.vm.provision "shell", inline: $linuxGvmProvision, privileged: false
+    c.vm.provision "shell", inline: linuxGvmProvision, privileged: false
```

Comment:
Does this need to be `linuxGvmProvision()`? (Not my area of expertise.)

Reply:
Because I've added a new optional parameter to define the architecture, GVM doesn't current use `i386` and I didn't want to add conditionals when we provision the machine on the bash scripts to correctly map the artifact to machine instead I just use the paremeters to correctly set the downloadable artifact.

Reply:
Just to clarify the above, `()` are not mandatory in ruby.

Reply:
Great. That's what I was wondering about. It looks like a function handle to me.

Reply:
Actually the 
```ruby
c.vm.provision "shell", inline: linuxGvmProvision, privileged: false
```

Is also a function call.
```ruby
c.vm.provision("shell", inline: linuxGvmProvision, privileged: false)
```
The following is also valid,

```ruby
c.vm.provision("shell", { inline: linuxGvmProvision, privileged: false })
```